### PR TITLE
Made statistic support green down arrows, and red up arrows, and grey…

### PR DESCRIPTION
**Pull request by SimplyLinn**
… arrows.

### CODE REVIEW

File: packages/next-dashboard/src/components/Statistic.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export type Props = {
   description?: R.Node,
   status?: Statuses,
   direction?: 'up' | 'down',
+  positive?: 'up' | 'down' | 'n/a',
```
Comment (SimplyLinn) [inline packages/next-dashboard/src/components/Statistic.js]:
```suggestion
  positive?: 'up' | 'down' | null,
```

Comment (SimplyLinn) [reply packages/next-dashboard/src/components/Statistic.js]: Look I'm tired and have been all day, I literally have no idea what I was thinking here.

Comment (FireyFly) [reply packages/next-dashboard/src/components/Statistic.js]: Does it make sense to have a `null` option here? Do we have a neutral type of up/down arrow? (like, grey or blue or something?)

Comment (SimplyLinn) [reply packages/next-dashboard/src/components/Statistic.js]: Yes, I added a grey arrow in the pier project.

File: packages/next-dashboard/src/components/Statistic.js [hunk 2]
```diff
@@ -31,14 +32,29 @@ function Statistic({
...
 }: Props): R.Node {
+  let isNegative = false;
+  let isPositive = false;
+  if (positive === 'down') {
+    isPositive = direction === 'down';
+    isNegative = direction === 'up';
+  } else if (positive === 'up') {
+    isPositive = direction === 'up';
+    isNegative = direction === 'down';
+  }
```
Comment (FireyFly) [inline packages/next-dashboard/src/components/Statistic.js]:
```js
const isPositive = (direction === positive);
const isNegative = (direction !== positive);
```
? I find it easier to reason about

Comment (SimplyLinn) [reply packages/next-dashboard/src/components/Statistic.js]:
```js
const positive = null;
const isNegative = down !== positive;
IsNegatve == true; // true
```
positive set to null (or n/a) means neither will be true, aka, neutral.

Comment (FireyFly) [reply packages/next-dashboard/src/components/Statistic.js]: Additionally.. I think I'd name the prop `positiveDirection` or something to make it a bit more clear that it's an enum and not a boolean. To me, `positive` and `negative` sound like boolean props (so you'd do something like `<Statistic positive value={10} />` or whatever)...

Minor thing, but I figure it doesn't hurt to suggest.. :)

Comment (SimplyLinn) [reply packages/next-dashboard/src/components/Statistic.js]:
```js
let isPositive = false;
let isNegative = false;
if(positive != null) {
 isPositive = (direction === positive);
 isNegative = (direction !== positive);
}
```
maybe in that case?

Comment (SimplyLinn) [reply packages/next-dashboard/src/components/Statistic.js]: Yeah, my brain has been barely functioning at all today, so nothing is named in a sane way...

_looks at branch name_

File: packages/next-dashboard/src/components/Statistic.js (hunk 1 above)
Comment (FireyFly) [reply packages/next-dashboard/src/components/Statistic.js]: Right, my bad, didn't see that! Yeah, this is good then :)

File: packages/next-dashboard/src/components/Statistic.js (hunk 2 above)
Comment (SimplyLinn) [inline packages/next-dashboard/src/components/Statistic.js]:
```suggestion
  let isNegative = false;
  let isPositive = false;
  if (positive != null) {
    isPositive = direction === positive;
    isNegative = direction === positive;
  }
```

Comment (FireyFly) [reply packages/next-dashboard/src/components/Statistic.js]: Ah, right! Or just `const isPositive = positive != null && direction === positive;` and conversely for `isNegative`. But I'm happy with `let`s and an `if` too!

**Pull request merged**